lib/make-middleware.js

```diff
@@ -138,7 +138,7 @@ function makeMiddleware (setup) {
 
         fileStream.on('limit', function () {
           aborting = true
-          abortWithCode('LIMIT_FILE_SIZE', fieldname)
+          abortWithCode('LIMIT_FILE_SIZE', fieldname, file)
```

Comment:
This won't do anything since `abortWithCode` only takes two arguments

Reply:
i´ll try to alter it when i got some spare time